src/main/java/book/store/bookstore/repositorie/impl/BookRepositoryImpl.java

```diff
@@ -0,0 +1,42 @@
+package book.store.bookstore.repositorie.impl;
+
+import book.store.bookstore.model.Book;
+import book.store.bookstore.repositorie.BookRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class BookRepositoryImpl implements BookRepository {
+    private final SessionFactory sessionFactory;
+
+    @Override
+    public Book save(Book book) {
+        Transaction transaction = null;
+        try (Session session = sessionFactory.openSession()) {
+            transaction = session.beginTransaction();
+            session.save(book);
+            transaction.commit();
+            return book;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can`t insert book into DB: " + book,e);
```

Comment:
You need space after "book," before commit code use Ctrl+Alt+L and Ctrl+Alt+O for cleaning code